packages/shared/src/components/Feed.tsx

```diff
@@ -199,6 +199,7 @@ export default function Feed<T>({
   const showScrollOnboardingVersion =
     sidebarRendered &&
     alerts?.filter &&
+    !user?.id &&
     Object.values(ScrollOnboardingVersion).includes(scrollOnboardingVersion);
   const fetchPageFn = showScrollOnboardingVersion
```

Comment:
I think we will need this since the new default value is string as well which would be true in this case.
```suggestion
  const fetchPageFn = showScrollOnboardingVersion === ScrollOnboardingVersion.Control
```

Reply:
I think it would be better to apply the condition in the `canFetchMore` property as well - should be along the lines below: https://github.com/dailydotdev/apps/blob/e8ce0cdd2083dae4ff6ca1f950bbdd9b00063a35/packages/shared/src/components/Feed.tsx#L204-L206

Because the `useInfiniteScroll` would probably think you are currently at the place where it should refetch, and might be executing the function endlessly.

```
const infiniteScrollRef = useFeedInfiniteScroll({
    fetchPage: fetchPageFn,
    canFetchMore: canFetchMore && showScrollOnboardingVersion !== ScrollOnboardingVersion.Control,
  });
```

Reply:
Didn't really look at this part of the code, but good catch.

Reply:
Adjusted it even further without `useState`. https://github.com/dailydotdev/apps/pull/1629/commits/d9a87c94125721999cc85e2cd048a3da7fff4ba6

Reply:
Yeah, just remembered it, sorry for having it requested now. Thank you for understanding 🙏

Reply:
No worries, it's definitely better this way.